client/settings/payment-methods-selector/index.js

```diff
@@ -10,12 +10,14 @@ import { useState, useCallback } from '@wordpress/element';
  * Internal dependencies
  */
 import {
+	useCurrencies,
 	useEnabledPaymentMethodIds,
 	useGetAvailablePaymentMethodIds,
-} from 'data';
+} from '../../data';
```

Comment:
The original `from 'data'` should still work right? It should resolves to `client/`. It makes no difference, but curious on the change. Also thinking if it's more consistent with the other places by using just `from 'data'`.

https://github.com/Automattic/woocommerce-payments/blob/4b8fd1bd27acc1232a0e3d4abd2078f11c19f586/webpack.config.js#L83

Reply:
I guess it applies also to all the `../../components` below too.

Reply:
Yeah it does, it's just a matter of preference :) 
```suggestion
} from 'data';
```